### ISSUE TITLE
What's the status of this project?

### ISSUE THREAD

**Issue by psyhomb** [TunSafe/TunSafe#68]
What's the status of this project?

**Comment by zt3ph4n** [comment]
Also like to know. *bump*

**Comment by zt3ph4n** [comment]
Suppose its dead, no answers nor updates?

**Comment by FlavioCFOliveira** [comment]
hello is anybody out there?

**Comment by nxxxse** [comment]
Happens. But that is not an issue as it is Free Software. We cannot demand somebody to continue, but we can do it ourselves. If everybody maintainers a few projects, we have everything we need.

**Comment by marfier** [comment]
> Happens. But that is not an issue as it is Free Software. We cannot demand somebody to continue, but we can do it ourselves. If everybody maintainers a few projects, we have everything we need.

I agree, but when open-source developers want to abandon a project, they normally announce it and archive the repository. It seems like the TunSafe developer just ghosted this, unless there's something we don't know yet.

**Comment by est** [comment]
This project needs a proper maintainer

**Comment by trinib** [comment]
this projects needs to be taken over.. any volunteers? ... it's the best VPN client out of a very few

**Comment by strongvpnok** [comment]
I have a working Windows version of `1.4.2` will all PRs merged. It also merged minor security patch from WireGuard. I uploaded it to gitea because i hate github for censoring users.

**Comment by strongvpnok** [comment]
If a'll you going to continue on github, good luck.


> this projects needs to be taken over.. any volunteers?

You really can't force people to do labour for free. Due to covid situation everyone is busy or don't have time. Donation or collaboration is always nice than demanding free slave work. This is why I build & patched it for my own use.

**Comment by psyhomb** [comment]
> I have a working Windows version of `1.4.2` will all PRs merged. It also merged minor security patch from WireGuard. I uploaded it to gitea because i hate github for censoring users.

Good for you.

There are other public git based services like for example [gitlab.com](https://gitlab.com) where you can upload your code, instead of using private gitea just because you hate github for censoring users. Yeah I don't like it either, but that's one of the reasons why I'm using [gitlab.com](https://gitlab.com) to make my code accessible to everyone.

> You really can't force people to do labour for free. Due to covid situation everyone is busy or don't have time. Donation or collaboration is always nice than demanding free slave work. This is why I build & patched it for my own use.

No one is forcing you to do anything, this is just a proposal and like I explained in previous sentence if you have already made some improvements and created a new release, it would be fair to [share](https://github.com/TunSafe/TunSafe/blob/master/LICENSE.AGPL.TXT) it at least.

**Comment by psyhomb** [comment]
There are other public git based services like for example gitlab.com where you can upload your code, instead of using private gitea just because you hate github for censoring users. Yeah I don't like it either, but that's why I'm using gitlab.com.

**Comment by tlsalex** [comment]
Also like to know

**Comment by tlsalex** [comment]
hey strigeus , where are you?

**Comment by VasiliyMooduckovich** [comment]
just use the [original ](https://download.wireguard.com/windows-client/)wireguard client for windows and forget about this

**Comment by Lockszmith-GH** [comment]
@VasiliyMooduckovich - while the original WireGuard is good, TunSafe does one thing particularly well - after it's installed by a local-admin, and admin isn't needed to switch on the VPN connection.

@strongvpnok - are you willing to share the link to your gitea repo? Are you maintaining installer builds?

**Comment by brendanosborne** [comment]
I am the creator of TunnlTo - a lightweight, fast, Windows WireGuard VPN client built for split tunneling. As this project appears to be abandoned, anyone who is looking for a Windows WireGuard client that provides more functionality over the standard WireGuard client might like to check out https://github.com/TunnlTo/desktop-app

**Comment by VasiliyMooduckovich** [comment]
> @VasiliyMooduckovich - while the original WireGuard is good, TunSafe does one thing particularly well - after it's installed by a local-admin, and admin isn't needed to switch on the VPN connection.

with original client it can be relatively easy done by adding a user to the "network operators" group and made some regedit tweaks, after that you don't have to be an administrator to just switch between tunnels more on this describe [here](https://arstechnica.com/gadgets/2020/11/wireguard-for-windows-0-3-1-is-the-release-youve-been-waiting-for/)
